pandas/tests/extension/base/ops.py

```diff
@@ -113,5 +114,5 @@ def test_compare_scalar(self, data, all_compare_operators):
     def test_compare_array(self, data, all_compare_operators):
         op_name = all_compare_operators
         s = pd.Series(data)
-        other = [0] * len(data)
+        other = pd.Series([data[0]] * len(data))
```

Comment:
why this? the point is to compare vs an array of things, NOT a Series of things.

Reply:
Not sure, but I had to make this same change on the SparseArray side of things. This eventually goes into `Series.combine(self, other)`, which doesn't support lists for `other`.

Reply:
In addition, I change here also `[0]` to `[data[0]]`, because `0` is not necessarily valid for all extension arrays

Reply:
so I had done this on purpose to test a list as an operand. So pls create an issue to add this back, to make sure we test list-like, array, Series.

Reply:
Opened https://github.com/pandas-dev/pandas/issues/22606